src/api/status/public/index.html

```diff
@@ -1222,6 +1284,7 @@ <h6 class="mt-3">Thank you for sharing!</h6>
     <script src="assets/js/plugins/chartjs.min.js"></script>
     <script src="assets/js/demo-chart.js"></script>
     <script src="assets/js/demo-sidenav.js"></script>
+    <script src="assets/js/gitHubStats.js"></script>
```

Comment:
Let's add a section specifically for our Telescope stuff

Reply:
inside `assets/js`? What should I call it?